lib/ui/home/home_page.dart

```diff
@@ -56,7 +56,8 @@ class HomePage extends HookConsumerWidget implements CobbleScreen {
       child: PlaceholderScreen(),
     ),
     _TabConfig(tr.homePage.store, RebbleIcons.rebble_store, child: StoreTab()),
-    _TabConfig(tr.homePage.watches, RebbleIcons.devices, child: MyWatchesTab()),
+    _TabConfig(tr.homePage.watches, RebbleIcons.devices,
+        onSelect: () => KMPApi().openWatchesView(), child: PlaceholderScreen()),
```

Comment:
My worry here is that if we disable the visibility of the old view, we lose quite a lot of functionality that came with it. We should really only do it if the functionality is caught up to the old view